File: leetcode/easy/1672/1672.rb
# Runtime: 52 ms, faster than 74.77% 
# Memory Usage: 209.8 MB, less than 96.73%
def maximum_wealth(accounts)
  max = 0

  accounts.each do |bank|
    sum = 0
    bank.each do |money|
      sum += money
    end
    max = sum > max ? sum : max
  end

  max
end

File: leetcode/easy/1672/1672.js
/**
 * Runtime: 80 ms, faster than 56.45% 
 * Memory Usage: 38.7 MB, less than 35.23%
 */
var maximumWealth = function(accounts) {
  let max = 0;
  accounts.forEach((bank) => {
    let sum = 0;
    bank.forEach((money) => {
      sum += money;
    });
    max = sum > max ? sum : max;
  });

  return max;
};
